-------------------------------------------------------------------------------
Test set: com.roshka.springboot.backend.apirest.SpringBootBackendApirestApplicationTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 5.743 s <<< FAILURE! - in com.roshka.springboot.backend.apirest.SpringBootBackendApirestApplicationTests
contextLoads  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'IClienteDao' defined in com.roshka.springboot.backend.apirest.models.dao.IClienteDao defined in @EnableJpaRepositories declared on JpaRepositoriesRegistrar.EnableJpaRepositoriesConfiguration: Cannot resolve reference to bean 'jpaMappingContext' while setting bean property 'mappingContext'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'jpaMappingContext': Invocation of init method failed; nested exception is org.hibernate.AnnotationException: mappedBy reference an unknown target entity property: com.roshka.springboot.backend.apirest.models.entity.Cabecera.usopuntos in com.roshka.springboot.backend.apirest.models.entity.UsoPuntos.cabeceras
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'jpaMappingContext': Invocation of init method failed; nested exception is org.hibernate.AnnotationException: mappedBy reference an unknown target entity property: com.roshka.springboot.backend.apirest.models.entity.Cabecera.usopuntos in com.roshka.springboot.backend.apirest.models.entity.UsoPuntos.cabeceras
Caused by: org.hibernate.AnnotationException: mappedBy reference an unknown target entity property: com.roshka.springboot.backend.apirest.models.entity.Cabecera.usopuntos in com.roshka.springboot.backend.apirest.models.entity.UsoPuntos.cabeceras

